src/wallet/wallet.h

```diff
@@ -700,7 +700,16 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
 
     bool fFileBacked;
 
-    std::set<int64_t> setKeyPool;
```

Comment:
Yes. But IMO holding a flexible data-structure (`KeypoolCache`) allows us to be more flexible in future. Though no strong opinion.